modules/platforms/dotnet/Apache.Ignite.Core.Tests/Compute/ComputeApiTest.cs

```diff
@@ -1155,7 +1163,7 @@ class ComputeAction : IComputeAction
 
         public static ConcurrentBag<Guid> Invokes = new ConcurrentBag<Guid>();
 
-        public static Guid LastNodeId;
+        public static ConcurrentDictionary<Guid, Guid> LastNodeId = new ConcurrentDictionary<Guid, Guid>();
```

Comment:
Naming is a bit confusing. Probably, should be something like "ParticipatedNodeIds" or something

Reply:
Good point, fixed.